### PR TITLE
Added a urdf example.

### PR DESCRIPTION
**Pull request by ugocupcic**
Had to create new fake calibration controller, very useful for plug and play experience using RoNeX.

### CODE REVIEW

**Comment by ugocupcic** [comment]
Do you think I should use an alias + tell people to set it in their parameter so that it's more plug and play?

File: sr_ronex_examples/config/joint_position_controller.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#Specifying the joint we want to be set to calibrated = true
+head_swivel_fake_calib:
+  type: sr_ronex_controllers/FakeCalibrationController
+  joint: head_swivel
+
+#Settings for a simple joint position controller
+head_swivel_controller:
+  type: robot_mechanism_controllers/JointPositionController
+  joint: head_swivel
+  pid: &head_swivel_gains
```
Comment (toliver) [inline sr_ronex_examples/config/joint_position_controller.yaml:10]: what does the & mean here?

**Comment by toliver** [comment]
About the alias, yes, it would be better with an alias.
I haven't used it in my examples, and although those are simpler and only require to edit one place in one file, I probably should make them use alias.
As the examples will be distributed as deb packages, either we instruct the users to overlay this package, or we provide a way to run the examples without having to modify them.

As you say, something like:
rosparam set /ronex/mapping/1234 "my_ronex"
roslaunch sr_ronex_examples the_example

File: sr_ronex_examples/config/joint_position_controller.yaml (hunk 1 above)
Comment (ugocupcic) [inline sr_ronex_examples/config/joint_position_controller.yaml:10]: I copy pasted from default pr2 controllers :s - apparently it's a reference (ie can use that param name multiple times)

see http://www.yaml.org/YAML_for_ruby.html#aliases_and_anchors

**Comment by ugocupcic** [comment]
Would be good to follow that principle for each of the tutorials.

**Comment by ugocupcic** [comment]
Wiki has been edited too.

**Pull request merged**